content/blog/animortis/_my_first_week_in_haiku.md

```diff
@@ -0,0 +1,39 @@
++++
+type = "blog"
+title = "My First Week in Haiku"
+author = "Animortis"
+date = "April 22, 2021"
+tags = ["haiku", "software"]
++++
+
+As you boot your first non-Windows, non-Unix system, you may have a moment when you wonder if this thing is going to work. It’s not from the same family of systems you’re used to: It’s Haiku, and it’s totally different and unique. The great part, though, is when it works better than you could’ve possibly imagined.
+
+And when I sat down with Haiku last week, I knew I had something special.
+
+Like a lot of people, I grew up using Windows, spent some time at work and in college on Macs and played with Linux off and on, watching the three OS’s dominate the market and define what computing could be for billions around the world. Even when I sat down with FreeBSD for the first time earlier this year, I knew more or less what I was doing already, since Linux commands worked (for the most part) almost exactly the same with it.
+
+How bland.
+
+But I had often heard about Haiku: The experimental successor to BeOS, an operating system that tech-guy-and-tv-host Leo Laporte used to show off on ZDTV in the 90s. Why, it could run *TWO* movies at once! And this was when even running one movie without a spectacular crash was an exceptional achievement for any computer! Even Windows 95 couldn’t do that! That blew my pre-teen mind.
+
+I was stunned to see Haiku install and boot faster than any *Nix or Windows systems could dream of. Pleasantly, it was also Posix compliant, too, meaning if I needed a quick fix for something then the command line was there waiting to help me out. It was ready to get things done.
+
+Haiku applications were so fast they practically screamed: The Mail app opened in a split second. The new video editor Medo natively handled 4K and offered multilingual translations. Plus, it also weighed in at about four megabytes and opened just as quickly.
+
+Haiku had all sorts of other programs available, too — everything from native, Haiku-only apps, utilities and games, to open-source favorites like LibreOffice, and tons of KDE apps brought in through HaikuPorts. These could be installed through its package manager, pkgman, or its built-in applications store, HaikuDepot, which were created by tireless volunteer developers from around the world.
+
+As for me, what I saw was enough to make me a believer: I decided to join the promotional team for Haiku. I wanted to bring this unique work of the technological arts to the world, to help people understand that they weren’t confined to using Linux all the time if they wanted free software. They could learn about something awesome, and even help turn it into the work of art it has the potential to be.
+
+Haiku can’t beat those mainstream behemoths without dedicated, passionate developers and volunteers. All operating systems start humbly and Haiku is no exception, even as it strives to become the first non-Unix open-source OS to hit version 1.0.
```

Comment:
"first non-Unix open-source OS to hit version 1.0" is a bit of a bold claim. It's not hard to find other OS that are already there, possibly in other markets. I'm thinking of several ones for embedded systems already.

In the desktop area there is AROS, Serenity, Redox, and probably many others.

Maybe we can instead mention we are one of the longest lived projects (not even the longest, AROS beats us by at least 5 years to that).

Reply:
That's fair. I did a lot of searching but never ran across any others, and even AROS is the only one I think really fits. Still, since we want accuracy, I tweaked it to "best" rather than "first."